101-bundle-json.md

```diff
@@ -769,8 +769,9 @@ Each action is accompanied by a description, which contains the following fields
 
 The `modifies` field MUST be set to `true` if any resource that is managed by the bundle is changed in any way. The `modifies` field assists CNAB implementations in tracking history of changes over time. An implementation of CNAB MAY use this information when describing history or managing releases.
 
-The `stateless` field indicates that the runtime bypass credentials validation (the user MAY or MAY NOT pass any credentials), and will not keep track of this action. Primary intent is to allow invocation bundles to provide dry-run or detailed help functionalities.
-Stateless actions can be invoked on a non-existing installation.
+The `stateless` field indicates that the runtime bypass credentials validation (the user MAY or MAY NOT pass any credentials), and will not keep track of this action. If credentials are supplied, the runtime SHOULD pass the credentials to the invocation image (as in non-stateless actions). An invocation image MAY exit with an error if the necessary credentials are not supplied. However, the runtime MUST NOT fail to invoke the invocation image because of missing credentials.
```

Comment:
the wording is off on this line: "The `stateless` field indicates that the runtime bypass credentials validation" . How about "The `stateless` field indicates that the runtime will bypass credential validation"

Reply:
consider changing "non-stateless actions" to "stateful actions"

Reply:
Made both changes. Thanks.